deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml

```diff
@@ -0,0 +1,1150 @@
+      extended_key_usage:
+      - client_auth
+      - server_auth
+- type: replace
+  path: /variables/-
+  value:
+    name: scheduler_ca
+    type: certificate
+    options:
+      common_name: schedulerCA
+      is_ca: true
+- type: replace
+  path: /variables/-
+  value:
+    name: scheduler_server
+    type: certificate
+    options:
+      ca: scheduler_ca
+      common_name: autoscalerscheduler.service.cf.internal
+      extended_key_usage:
+      - client_auth
+      - server_auth
+- type: replace
+  path: /variables/-
+  value:
+    name: scheduler_client
+    type: certificate
+    options:
+      ca: scheduler_ca
+      common_name: scheduler client
+      extended_key_usage:
+      - client_auth
+- type: replace
+  path: /variables/-
+  value:
+    name: metricsserver_ca
+    type: certificate
+    options:
+      common_name: metricsserverCA
+      is_ca: true
+- type: replace
+  path: /variables/-
+  value:
+    name: metricsserver_server
+    type: certificate
+    options:
+      alternative_names:
+      - metricsserver.service.cf.internal
+      - '*.asmetrics.default.app-autoscaler.bosh'
+      ca: metricsserver_ca
+      common_name: metricsserver.service.cf.internal
+      extended_key_usage:
+      - client_auth
+      - server_auth
+- type: replace
+  path: /variables/-
+  value:
+    name: metricsserver_client
+    type: certificate
+    options:
+      ca: metricsserver_ca
+      common_name: metricsserver client
+      extended_key_usage:
+      - client_auth
+- type: replace
+  path: /variables/-
+  value:
+    name: postgres_ca
+    type: certificate
+    options:
+      common_name: postgresCA
+      is_ca: true
+- type: replace
+  path: /variables/-
+  value:
+    name: postgres_server
+    type: certificate
+    options:
+      ca: postgres_ca
+      common_name: autoscalerpostgres.service.cf.internal
+      extended_key_usage:
+      - client_auth
+      - server_auth   
+
+# Set the correct kube addresses
+- type: replace
+  path: /instance_groups/name=asnozzle/jobs/name=metricsgateway/properties/autoscaler/metricsgateway/nozzle/rlp_addr?
+  value: {{ .Values.deployment_name }}-log-api:8082
+- type: replace
+  path: /variables/name=eventgenerator_server/options?/alternative_names
+  value: ["{{ .Values.deployment_name }}-asmetrics","*.{{ .Values.deployment_name }}-asmetrics","{{ .Values.deployment_name }}-asmetrics-0","{{ .Values.deployment_name }}-asmetrics-1"]
+- type: replace
+  path: /variables/name=metricsserver_server/options?/alternative_names
+  value: ["{{ .Values.deployment_name }}-asmetrics","*.{{ .Values.deployment_name }}-asmetrics","{{ .Values.deployment_name }}-asmetrics-0","{{ .Values.deployment_name }}-asmetrics-1"] 
+
+- type: remove
+  path: /releases/name=app-autoscaler
+- type: remove
+  path: /releases/name=postgres
+- type: replace
+  path: /releases/-
+  value:
+    name: app-autoscaler
+    version: "2.0.0"
+    url: docker.io/aqan
```

Comment:
The `url` and `stemcell` values should come from `values.yaml`. Take a look on https://github.com/SUSE/kubecf/pull/129 for how we are doing it.

Reply:
I have changed the `url` and `stemcell` values and added the `releases.autoscaler.stemcell` in `values.yaml`.  
```
releases:
  autoscaler:
    stemcell:
      version: 38.g82067a9-30.95-7.0.0_346.ge9dd9ff3
```
But the image `cfcontainerization/app-autoscaler:opensuse-42.3-38.g82067a9-30.95-7.0.0_346.ge9dd9ff3-2.0.0` and `cfcontainerization/app-autoscaler:opensuse-42.3-38.g82067a9-30.95-7.0.0_346.ge9dd9ff3-2.0.0` do not exist at the moment since I have no access for the image org.

Reply:
We will add it to our pipeline. It should be available soon.

Reply:
https://github.com/SUSE/kubecf/issues/34

Reply:
Thanks for providing the image.